packages/lesswrong/lib/collections/posts/views.ts

```diff
@@ -284,6 +284,11 @@ ensureIndex(Posts,
     name: "posts.score",
   }
 );
+
+
+// Wildcard index on tagRelevance, enables us to efficiently filter on tagRel scores
+// EA-FORUM: To run this you will have to update to MongoDB 4.2
```

Comment:
We used to have an index that would fail to build (it was indexing the `bio` field on users, which is too big), and it would produce an error message on startup but it was mostly harmless. My guess is that running this on a too-old mongodb, when you aren't actually using tagging, is probably harmless in the same way.

Reply:
```suggestion
// EA-FORUM: Building this index will fail until you update to MongoDB 4.2. If you haven't enabled/started using tagging, then this is probably harmless.
```